packages/doenetml-worker-rust/lib-doenetml-core/src/core/state/types/list_marker.rs

```diff
@@ -0,0 +1,214 @@
+/// A list marker. This is used by both `<ol>` and `<ul>` elements.
+/// They are named similarly to their [HTML equivalents](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type).
+#[derive(Debug, Clone, Copy, PartialEq, serde::Serialize, serde::Deserialize)]
+#[cfg_attr(feature = "web", derive(tsify_next::Tsify))]
+pub enum ListMarker {
+    // Ordered list
+    /// Label with a number, starting at `start`. e.g. `1`, `2`, `3`, ...
+    Decimal { start: u8 },
```

Comment:
I am confused as I cannot find the use of the value of this `start` field anywhere. Does it do anything?

Reply:
It gets used as long as the `start` attribute is not used.

Reply:
OK, it's just information about the list marker that can be used by outside consumers.